numbers.qmd

```diff
@@ -30,6 +30,27 @@ library(tidyverse)
 library(nycflights13)
 ```
 
+## Making numbers
+
+In most cases, you'll get numbers already recorded in one R's numeric types, integer or double.
+In some cases, however, you'll encounter them as strings, possibly because you've created the by pivotting from column headers or something has gone wrong in your data import process.
```

Comment:
```suggestion
In some cases, however, you'll encounter them as strings, possibly because you've created them by pivoting from column headers or something has gone wrong in your data import process.
```